src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,42 +1,82 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { currentTodoSlice } from '../../features/currentTodo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
 
 export const TodoModal: React.FC = () => {
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const dispatch = useAppDispatch();
+
+  const [user, setUser] = useState<User | null>();
```

Comment:
The `useState` for `user` should be initialized with `null` as its initial value: `useState<User | null>(null);` This ensures that the state is explicitly set to `null` initially.